src/App.js

```diff
@@ -1,14 +1,22 @@
-import { Component } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
-class App extends Component {
-  render() {
-    return (
+function App() {
+  return (
+    <Router>
       <div className="App">
-        <Calculator />
+        <Navbar />
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route exact path="/calculator" element={<Calculator />} />
+          <Route exact path="/quote" element={<Quotes />} />
+        </Routes>
       </div>
-    );
-  }
+    </Router>
+  );
 }
 
 export default App;
```

Comment:
- [ ] As it is a requirement for this project, and best practice to stick to the wireframe as much as possible, please kindly update your app to match the wireframe. See screenshots below and more clarifications.

***The wireframe***
![theWireframe](https://user-images.githubusercontent.com/65852621/156156150-fe3db906-255a-485f-a853-a693d3a3b9b3.png)

***Your implementation***
![yourImplementation](https://user-images.githubusercontent.com/65852621/156156187-2e87fd65-54e1-4beb-9174-6902a5397427.png)